nuxeo-common/src/main/java/org/nuxeo/common/xmap/XValueFactory.java

```diff
@@ -220,6 +223,42 @@ public String serialize(Context context, Object value) {
         }
     };
 
+    public static final XValueFactory DURATION = new XValueFactory() {
+        public final Pattern durationSimpleFormat = Pattern.compile("(?:(\\d+)m)?(?:(\\d+)s)?(?:(\\d+)ms)?");
```

Comment:
Why not also include days and hours, to try to be more complete and comparable to Duration?